### ISSUE TITLE
changeState instead of file

### ISSUE THREAD

**Issue by h1alexbel** [eo-cqrs/cmig#32]

**Comment by zoeself** [comment]
@h1alexbel thank you for reporting this. I'll assign someone to take care of it soon.

**Comment by zoeself** [comment]
@h1alexbel this is your task now, please go ahead. Deadline (when this ticket should be closed) is ``2023-08-20T18:59:15.027817``.

Estimation here is ``60 minutes``, that's how much you will be paid.

Remember, you don't have to solve everything in this ticket - you can solve it [partially](https://docs.self-xdsd.com/microtasking.html) and leave ``todo`` markers in the code, which will become future tasks.

If you have any questions don't ask me, I'm not a technical person. Open new tickets instead.

If you don't have time or simply don't want to work on this, you can always [resign](https://docs.self-xdsd.com/projectmanager.html#commands).

**Comment by h1alexbel** [comment]
```xml
<states>
  <changeState id="1" author="test">
    <files>
      <file path="001-initial-keyspace.cql"/>
      <file path="002-queries-table.cql"/>
    </files>
  </changeState>
</states>
```

**Comment by 0pdd** [comment]
@h1alexbel 2 puzzles [#36](https://github.com/eo-cqrs/cmig/issues/36), [#37](https://github.com/eo-cqrs/cmig/issues/37) are still not solved.

**Comment by zoeself** [comment]
@h1alexbel thank you for resolving this ticket. I've just added it to your active invoice. You can always check all your invoices and more on the [Contributor Dashboard](https://self-xdsd.com).

**Comment by 0pdd** [comment]
@h1alexbel 4 puzzles [#36](https://github.com/eo-cqrs/cmig/issues/36), [#37](https://github.com/eo-cqrs/cmig/issues/37), [#47](https://github.com/eo-cqrs/cmig/issues/47), [#48](https://github.com/eo-cqrs/cmig/issues/48) are still not solved.

**Comment by 0pdd** [comment]
@h1alexbel 5 puzzles [#36](https://github.com/eo-cqrs/cmig/issues/36), [#47](https://github.com/eo-cqrs/cmig/issues/47), [#48](https://github.com/eo-cqrs/cmig/issues/48), [#49](https://github.com/eo-cqrs/cmig/issues/49), [#50](https://github.com/eo-cqrs/cmig/issues/50) are still not solved; solved: [#37](https://github.com/eo-cqrs/cmig/issues/37).

**Comment by 0pdd** [comment]
@h1alexbel 4 puzzles [#36](https://github.com/eo-cqrs/cmig/issues/36), [#47](https://github.com/eo-cqrs/cmig/issues/47), [#48](https://github.com/eo-cqrs/cmig/issues/48), [#49](https://github.com/eo-cqrs/cmig/issues/49) are still not solved; solved: [#37](https://github.com/eo-cqrs/cmig/issues/37), [#50](https://github.com/eo-cqrs/cmig/issues/50).

**Comment by 0pdd** [comment]
@h1alexbel 4 puzzles [#36](https://github.com/eo-cqrs/cmig/issues/36), [#49](https://github.com/eo-cqrs/cmig/issues/49), [#64](https://github.com/eo-cqrs/cmig/issues/64), [#65](https://github.com/eo-cqrs/cmig/issues/65) are still not solved; solved: [#37](https://github.com/eo-cqrs/cmig/issues/37), [#47](https://github.com/eo-cqrs/cmig/issues/47), [#48](https://github.com/eo-cqrs/cmig/issues/48), [#50](https://github.com/eo-cqrs/cmig/issues/50).

**Comment by 0pdd** [comment]
@h1alexbel 4 puzzles [#36](https://github.com/eo-cqrs/cmig/issues/36), [#49](https://github.com/eo-cqrs/cmig/issues/49), [#64](https://github.com/eo-cqrs/cmig/issues/64), [#98](https://github.com/eo-cqrs/cmig/issues/98) are still not solved; solved: [#37](https://github.com/eo-cqrs/cmig/issues/37), [#47](https://github.com/eo-cqrs/cmig/issues/47), [#48](https://github.com/eo-cqrs/cmig/issues/48), [#50](https://github.com/eo-cqrs/cmig/issues/50), [#65](https://github.com/eo-cqrs/cmig/issues/65).
